src/components/PageTitle.tsx

```diff
@@ -1,6 +1,7 @@
 import styled from "@emotion/styled";
 import { H2, Subtitle } from "@leafygreen-ui/typography";
 import { Skeleton } from "antd";
```

Comment:
Is there a bug with the badge placement? It's normally right after & inline with the title

**in this PR:**
<img width="1052" alt="Screen Shot 2022-02-08 at 4 05 39 PM" src="https://user-images.githubusercontent.com/47064971/153075501-1045641a-39ed-41b7-9b2d-9e5d042d3f45.png">

<img width="1436" alt="Screen Shot 2022-02-08 at 4 23 00 PM" src="https://user-images.githubusercontent.com/47064971/153078270-fe7abe60-7b12-43ad-9f6c-3da272203243.png">



**normally:**
<img width="929" alt="Screen Shot 2022-02-08 at 4 06 13 PM" src="https://user-images.githubusercontent.com/47064971/153075583-4ec51957-186a-4743-97e9-9e13b979f476.png">

Reply:
Great catch!